Makefile

```diff
@@ -311,3 +311,53 @@ build-chart-ttl.sh: export OPERATOR_IMAGE_TAG = 24h
 build-chart-ttl.sh: export CHART_REMOTE = oci://ttl.sh/${CURRENT_USER}
 build-chart-ttl.sh:
 	cd charts/embedded-cluster-operator && ../../scripts/publish-helm-chart.sh
+
+.PHONY: apko-build
+apko-build: export IMAGE = ttl.sh/${CURRENT_USER}/embedded-cluster-operator-image:24h
```

Comment:
wouldn't this always override the environment variables?

Reply:
yes. fixing